app/routes/markets/index.js

```diff
@@ -79,4 +79,9 @@ router.get('/:marketKey', (req, res, next) => {
   });
 });
 
+router.post('/delete', (req, res, next) => {
```

Comment:
Sorry should've caught this sooner but instead of `POST /delete` with the name in the body, use `DELETE /:market`; that's a more RESTful route since the HTTP method is the verb and the URI corresponds to the relevant resource. A good rule of thumb when designing REST routes is to avoid verbs in URIs wherever possible (and it's almost always possible).